src/main/java/Contacts0.java

```diff
@@ -3,14 +3,13 @@
 
 public class Contacts0 {
 
+    private static final int MAXIMUM_CAPACITY = 100;
+    private static final int NUMBER_OF_FIELDS = 3;
+
     public static void main(String[] args) {
         final Scanner SCANNER = new Scanner(System.in);
-        System.out.println("|| ===================================================");
-        System.out.println("|| ===================================================");
-        System.out.println("|| Contacts - Version 0.0");
-        System.out.println("|| Welcome to Contacts!");
-        System.out.println("|| ===================================================");
-        String[][] list = new String[100][3];
+        displayWelcomeMessage();
+        String[][] list = new String[MAXIMUM_CAPACITY][NUMBER_OF_FIELDS];
```

Comment:
Very nice refactoring of the code! Now it looks neater and easier to understand :+1: